Source/Core/Core/IOS/DolphinDevice.cpp

```diff
@@ -0,0 +1,81 @@
+// Copyright 2019 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include <sys/time.h>
+
+#include "Core/HW/Memmap.h"
+#include "Core/IOS/DolphinDevice.h"
+#include "Common/Logging/Log.h"
+#include "Common/scmrev.h"
+#include "Common/Timer.h"
+
+
+namespace IOS::HLE::Device
+{
+  
+  IPCCommandResult GetSystemTime(const IOCtlVRequest& request);
```

Comment:
There's a misunderstanding here when I meant they could be moved into the cpp file since they're implementation details. I was talking about the enum constants, not the actual function prototypes.

Reply:
They are implementation details? For /dev/es they are in the .h file as well (which I used as a "base"). Why would these be implementation details if custom code would need to know these to access the appropriate function?

Reply:
If they aren't used outside of the es code, they should also be in the cpp file as well. They're literally only used within `IOCtlV` inside the translation unit and not outside of it. By definition they're implementation details relative to other constructs within the codebase.

If other people need to know about things in particular, this is better served through a direct documentation comment rather than making reading code a requirement.

Reply:
Okay, I'll move them to the cpp file.